protocol/x/clob/abci.go

```diff
@@ -61,9 +61,10 @@ func EndBlocker(
 		keeper.DeleteLongTermOrderPlacement(ctx, orderId)
 
 		// Emit an on-chain indexer event for Stateful Order Expiration.
-		keeper.GetIndexerEventManager().AddTxnEvent(
+		keeper.GetIndexerEventManager().AddBlockEvent(
```

Comment:
this is the comment in their commit cant explain why dont understand indexer that well: "Add stateful order removals as block events rather than transaction events to fix their ordering in the produced indexer block message."

Reply:
ok I think we include this in the checklist when we do deep dive in indexer